assignments/array-methods.js

```diff
@@ -56,28 +56,63 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
 // ==== Challenge 1: Use .forEach() ====
 // The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName. 
 let fullName = [];
+/* for (let i=0; i<runners.length; i++){
+    let firstName = runners[i].first_name;
+    let lastName = runners[i].last_name;
+    fullName.push(`${firstName} ${lastName}`); 
+} */
+runners.forEach(r => fullName.push(`${r.first_name} ${r.last_name}`))
 console.log(fullName);
 
 // ==== Challenge 2: Use .map() ====
 // The event director needs to have all the runner's first names converted to uppercase because the director BECAME DRUNK WITH POWER. Convert each first name into all caps and log the result
 let allCaps = [];
+allCaps = runners.map(function(lowercase){
+    return lowercase.first_name.toUpperCase();
+})
```

Comment:
Converted to arrow function:
`allCaps = runners.map(runner=> runner.first_name.toUpperCase())`